app/public/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
```

Comment:
Oh, also this includes the react favicon. Can we just delete that file for now? We'll need a new packit logo I guess! Could you add a ticket for designing a logo and creating a favicon?